.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        default: "0.0.1"
+        description: "Version of the release"
+        required: true
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ inputs.version }}
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: configure git
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email '41898282+github-actions[bot]@users.noreply.github.com'
+      - name: bump versions
+        run: |
+          DATE=$(date +"%B %d %Y")
+          MANPAGE=".TH wd \"1\" \"$DATE\" \"wd $VERSION\" \"wd Manual\""
+          sed -i -E "1 s/.*/$MANPAGE/" wd.1
+          sed -i -E "s/WD_VERSION=.*/WD_VERSION=$VERSION/" wd.sh
+      - name: commit changes
+        run: |
+          git add wd.1 wd.sh
+          git commit -m "chore(update): bump current version to v$VERSION"
+          git tag v$VERSION
+          git push -u origin master --tags
+      - name: release latest
+        run: |
+          gh release create v$VERSION --generate-notes
+        env:
```

Comment:
Is this not set up automatically? I would have expected it to be

Reply:
The `GITHUB_TOKEN` environment variable or `github.token` value might be set by default, but [the docs suggest](https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows) needing `GH_TOKEN` with the `gh` CLI:

> For each step that uses GitHub CLI, you must set an environment variable called `GH_TOKEN` to a token with the required scopes.

I'll also test this - with better quoting for the tag - to see if it works without the `env`!